Sources/SwiftGRPCNIO/GRPCChannelHandler.swift

```diff
@@ -55,9 +55,8 @@ extension GRPCChannelHandler: ChannelInboundHandler {
           return
       }
 
-      var responseHeaders = HTTPHeaders()
-      responseHeaders.add(name: "content-type", value: "application/grpc")
-      ctx.write(self.wrapOutboundOut(.headers(responseHeaders)), promise: nil)
+      // Send empty headers to start the outbound header workflow.
```

Comment:
Would you mind explaining this change in more detail?

Reply:
Added more, please let me know if it makes sense. I tried modifying the architecture so that the headers are sent from `HTTP1ToRawGRPCServerCode`, but it was getting to complex.

Reply:
Hmm, is it possible to send multiple header frames in HTTP1 without problems?